public/index.html

```diff
@@ -5,7 +5,7 @@
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="description" content="Web site created using create-react-app" />
-    <title>React App</title>
+    <title>AnJu's Ghibli App</title>
```

Comment:
I love that you added this little touch!

Reply:
:-)